runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessConfigureView/ProcessConfigureView.tsx

```diff
@@ -89,8 +121,21 @@ const ProcessConfigureView: VFC = () => {
         await fetchProcess();
     };
 
+    const handleSubscriptionChange = async (subscriptionState: boolean) => {
+        subscriptionState
+            ? await dispatch(processActions.subscribeProcessNotifications({ user, process }))
+            : await dispatch(processActions.unsubscribeProcessNotifications(currentProcessSubscription.id));
+
+        await handleGetProcessSubscribers();
+    };
+
+    async function handleDeleteSubscription(subscriptionInfo: ProcessNotificationRow) {
+        await dispatch(processActions.unsubscribeProcessNotifications(subscriptionInfo.id));
+        await handleGetProcessSubscribers();
+    }
```

Comment:
imo if there is no connection correlation between those requests then make it parallel

Reply:
there are many more similar functions in this PR